### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by kisszoe2017** [PasteBar/PasteBarApp#187]
Please consider adding more options. When I use the global hotkey, the popup window disappears, lol. The reason is that its popup is not fixed, and on a large screen, it can be hard to find its position sometimes. Please consider making its appearance fixed in the next version.

There also seems to be a slight delay when invoking the paste window with the hotkey, which might be due to having too many history entries. Please consider adding an option to limit the maximum number of saved history entries. I am also looking forward to the addition of an option to limit the number of lines per clipboard entry.

I want to express my gratitude for this application. It has greatly benefited my work, and your dedication to maintaining it is commendable. Thank you very much.

**Comment by David-design352** [comment]
Bro, it will appear near your mouse, won't it?

**Comment by kurdin** [comment]
@kisszoe2017 
As @David-design352 suggested, the Quick Paste popup window will be open close to your current cursor position. Please show me a video where you can demonstrate problem locating the window on global hotkey press. 

I have 2 monitors and have never had a problem with the Quick Paste popup window visibility. It works pretty consistently for me at least on my Mac. 

> I want to express my gratitude for this application. It has greatly benefited my work, and your dedication to maintaining it is commendable. Thank you very much.
Thank you very much!

**Comment by kisszoe2017** [comment]
Yes, sir, you are absolutely right. Previously, I didn’t realize that it appears near the cursor (the reason I couldn’t find it was that my mouse sometimes moved to the edge of the screen, and my English proficiency is not very high, so it took me some time to locate the Simplified Chinese version). Please consider adding a feature to remember the last popup position.

**Comment by IchHabRecht** [comment]
Hi @kurdin ,

I can confirm a hidden Quick Paste popup on Windows. I'm working with three screens and opening the popup with global hotkey each time on a different screen will break after a short while. I try to cast a video about that the next couple of days.

**Comment by kurdin** [comment]
implemented #248
